### PR TITLE
docs(project): add note on audit fixes

### PR DESCRIPTION
**Pull request by nikku**

### CODE REVIEW

**Review by barmac — APPROVED** [review]

**Comment by barmac** [comment]
Let's add it to https://github.com/camunda/camunda-modeler/blob/ea73385ec8a151b95f876f9cbb3b11776637e887/docs/.project/PATCH_RELEASE_TEMPLATE.md too

File: docs/.project/RELEASE_TEMPLATE.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ _To be done after code freeze to prepare and test the release._
 * [ ] make sure dependencies to upstream libraries are updated and can be installed (`rm -rf node_modules && npm i && npm run all` works)
 * [ ] verify `develop` is up to date with `master`: `git checkout master && git pull && git checkout develop && git merge master`
 * [ ] close all issues which are solved by dependency updates
+* [ ] Ensure that the modeler is free of major security vulnerabilities via `npm audit`
```
Comment (marstamm) [inline docs/.project/RELEASE_TEMPLATE.md:36]: let's keep the capitalization consistent

```suggestion
* [ ] ensure that the modeler is free of major security vulnerabilities via `npm audit`
```

**Comment by marstamm** [comment]
I also added it to the current Issue: https://github.com/camunda/camunda-modeler/issues/3919

**Review by marstamm — APPROVED** [review]

**Comment by nikku** [comment]
@barmac Added to patch via https://github.com/camunda/camunda-modeler/pull/3930/commits/72902dcae05d32b46b4ff0a31f44dbec64353bc9, too.

**Pull request merged**